user-declaration-util/src/main/java/org/sunbird/migration/RecordProcessor.java

```diff
@@ -168,10 +168,18 @@ private void updateUserSelfDeclareLists(
             userInfo.put(key, value);
           }
           if (userDeclareEntity.getStatus() == null || userDeclareEntity.getStatus().isEmpty()) {
-            userDeclareEntity.setStatus("PENDING");
+            userDeclareEntity.setStatus("SUBMITTED");
             characterCount.put(
-                "PENDING",
-                characterCount.get("PENDING") == null ? 1 : characterCount.get("PENDING") + 1);
+                "SUBMITTED",
+                characterCount.get("SUBMITTED") == null ? 1 : characterCount.get("SUBMITTED") + 1);
+            removeFlag = false;
+          }
+          if (userDeclareEntity.getStatus() != null
+              && "PENDING".equals(userDeclareEntity.getStatus())) {
+            userDeclareEntity.setStatus("SUBMITTED");
+            characterCount.put(
```

Comment:
We may not be interested in this count - it is adding to the same counter. This is important to know if there is leakage in our codes that allow status to be empty.
 
This if-part is true to be all records at least the first run after this PR goes live. So, suggest you to remove. We can have another counter separately. And this counter must be 0 after the first run.

Reply:
ok done